platform-includes/user-feedback/sdk-api-example/react-native.mdx

```diff
@@ -1,16 +1,35 @@
 ```typescript
 import * as Sentry from "@sentry/react-native";
-import { UserFeedback } from "@sentry/react-native";
+import { SendFeedbackParams } from "@sentry/react-native";
 
 const sentryId = Sentry.captureMessage("My Message");
 // OR: const sentryId = Sentry.lastEventId();
 
-const userFeedback: UserFeedback = {
-  event_id: sentryId,
+const userFeedback: SendFeedbackParams = {
   name: "John Doe",
   email: "john@doe.com",
-  comments: "Hello World!",
+  message: "Hello World!",
+  associatedEventId: eventId,// Optional
 };
 
 Sentry.captureUserFeedback(userFeedback);
```

Comment:
We should not show the old api anywhere after the new `captureFeedback` is live.

````suggestion
Sentry.captureFeedback(userFeedback);
````